src/sparseml/pytorch/model_load/helpers.py

```diff
@@ -39,6 +39,45 @@
 RECIPE_FILE_NAME = "recipe.yaml"
 
 
+def log_model_load(
```

Comment:
This is just moving `log_model_load` from the `SparseModel` class (no reason why it should be folded under that class really, should be a standalone)

Reply:
nice!